docs/docusaurus.config.js

```diff
@@ -83,6 +83,7 @@ module.exports = {
 			'@docusaurus/preset-classic',
 			{
 				docs: {
+					homePageId: 'index',
```

Comment:
This option negates the need for an index.js file?

Reply:
yep. It was a new feature that was just added